src/OpenSage.Game/Game.cs

```diff
@@ -532,7 +532,7 @@ public Game(GameInstallation installation)
                 OrderGenerator = AddDisposable(new OrderGeneratorSystem(this));
 
                 Panel.ClientSizeChanged += OnPanelSizeChanged;
-                OnPanelSizeChanged(this, EventArgs.Empty);
+                Panel.EnsureFrame(window.ClientBounds);
```

Comment:
Setting the frame size here ensures we never attempt to construct a camera with 0 height (and thus a NaN FOV)